ts/features/pn/screens/LegacyMessageDetailsScreen.tsx

```diff
@@ -0,0 +1,140 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { pipe } from "fp-ts/lib/function";
+import * as O from "fp-ts/lib/Option";
+import React from "react";
+import { SafeAreaView } from "react-native";
+import { useFocusEffect, useNavigation } from "@react-navigation/native";
+import { useStore } from "react-redux";
+import { IOStyles } from "../../../components/core/variables/IOStyles";
+import BaseScreenComponent from "../../../components/screens/BaseScreenComponent";
+import { ServiceId } from "../../../../definitions/backend/ServiceId";
+import I18n from "../../../i18n";
+import { IOStackNavigationRouteProps } from "../../../navigation/params/AppParamsList";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { UIMessageId } from "../../messages/types";
+import { serviceByIdSelector } from "../../../store/reducers/entities/services/servicesById";
+import { emptyContextualHelp } from "../../../utils/emptyContextualHelp";
+import { useOnFirstRender } from "../../../utils/hooks/useOnFirstRender";
+import { LegacyMessageDetails } from "../components/LegacyMessageDetails";
+import { PnParamsList } from "../navigation/params";
+import { pnMessageFromIdSelector } from "../store/reducers";
+import { cancelPreviousAttachmentDownload } from "../../messages/store/actions";
+import { profileFiscalCodeSelector } from "../../../store/reducers/profile";
+import {
+  containsF24FromPNMessagePot,
+  isCancelledFromPNMessagePot,
+  paymentsFromPNMessagePot
+} from "../utils";
+import { trackPNUxSuccess } from "../analytics";
+import { isStrictSome } from "../../../utils/pot";
+import {
+  cancelPaymentStatusTracking,
+  cancelQueuedPaymentUpdates,
+  clearSelectedPayment,
+  startPaymentStatusTracking,
+  updatePaymentForMessage
+} from "../store/actions";
+import { GlobalState } from "../../../store/reducers/types";
+import { selectedPaymentIdSelector } from "../store/reducers/payments";
+import { InfoScreenComponent } from "../../../components/infoScreen/InfoScreenComponent";
+import { renderInfoRasterImage } from "../../../components/infoScreen/imageRendering";
+import genericErrorIcon from "../../../../img/wallet/errors/generic-error-icon.png";
+
+export type MessageDetailsScreenNavigationParams = Readonly<{
```

Comment:
Should we remove this type and rely on the new screen only?

Reply:
Applied https://github.com/pagopa/io-app/pull/5431/commits/1bee02ac06a5964aa3d0497341ec972fc7cb1d14